package.json

```diff
@@ -41,8 +41,8 @@
     "test:unit:watch": "cross-env NODE_ENV=test babel-node test/watch.js unit"
   },
   "peerDependencies": {
-    "react": "^15.4.0",
-    "react-dom": "^15.4.0",
+    "react": "^15.5.0",
+    "react-dom": "^15.5.0",
```

Comment:
You are referring to the following `dependency`
```
+    "react-addons-create-fragment": "^15.5.0",
+    "react-addons-transition-group": "^15.5.0",
```

Same, I don't think that we need to update those. If we do, let's release a `minor`.